lib/organization.js

```diff
@@ -86,9 +85,17 @@ export class Organization {
     org.reputation = await Reputation.at(reputationAddress);
 
     org.votingMachine = await AbsoluteVote.at(options.votingMachine);
```

Comment:
Do we need AbsoluteVote explicitly here.
The voting machine should be pass as a param for each scheme (if this scheme need voting machine as param).